src/sass/card/card.scss

```diff
@@ -0,0 +1,135 @@
+@import "../mixins/private/bubble-mixins";
+@import "../mixins/private/token-mixins";
+@import "../variables/variables";
+
+.card {
+    display: flex;
+    flex-direction: column;
+    width: $card-size-default;
+}
+
+.card__container {
+    position: relative;
+    height: $card-size-default;
```

Comment:
Because the card can contain details, that's why the width and height are separate so that the width can be applied to the details as well since the card is the parent of the card container and the card details. Sounds like the size should be adjustable?